Include guidance on sharing content (#2117)
writethedocs/www: docs/slack.rst

- **Avoid technology competitions.**
  It doesn't matter if your tool is faster or better in some metric.
  We care about creating great documentation for our users.
- **Respect the Community's Privacy**
  Do not share WtD Slack content externally without permission from participants of that discussion. 

>   Think of it like the `Chatham House Rule <https://en.wikipedia.org/wiki/Chatham_House_rule>`, and either ask for permission or sufficiently anonymize the content where no one could reasonably guess the author or source.

Author: ```suggestion
  Think of it like the `Chatham House Rule <https://en.wikipedia.org/wiki/Chatham_House_rule>`, and *publicly* ask for permission from the participants *or*  anonymize the content where no one could reasonably guess the author(s) or source.
```

Potential alternative

>   Think of it like the `Chatham House Rule <https://en.wikipedia.org/wiki/Chatham_House_rule>`, and either ask for permission or sufficiently anonymize the content where no one could reasonably guess the author or source.

Reviewer: ```suggestion
  Think of it like the `Chatham House Rule <https://en.wikipedia.org/wiki/Chatham_House_rule>`, and either ask for permission or anonymize the content.
```

if you prefer more straightfoward

Author: I like straightforward.

>   Violating this rule may lead to an immediate ban of your account.

Author: ```suggestion
  It is your responsibility, not the participants, to ensure that everyone has consented to your sharing of the content.
```

In case the above is too stiff.

Reviewer: I like this change, although "it is your responsibility" ...

> - **Respect the privacy of your fellow community members.**
>   We love it when people get inspired by the conversations they have in the Write the Docs Slack! When doing so, make sure you follow best practices when sharing information about those conversations in things like blog and LinkedIn posts.
>   Make sure you always get permission in a public channel from participants in a discussion before sharing anything with any names attached. It is your responsibility, not the participants', to ensure that everyone has consented to be named in any external publications.
>   Otherwise, you can use the `Chatham House Rule <https://en.wikipedia.org/wiki/Chatham_House_rule>`_ as a guideline, and anonymize the content where no one could reasonably guess the author(s) or source.

Reviewer 2: Invoking privacy here is not working for me: we're not making a privacy policy and I worry that we run the risk of putting participants at extra risk by giving a false expectation of privacy where one cannot exist.

Instead, I think we should frame this as a tool for having valuable converstions. Something like this (roughly, not expecting you take this verbatim):

```suggestion
- **Honor the community's trust and confidence.**
  In the Write the Docs Slack, we are privileged to be able to discuss complex or sensitive situations with other docs practitioners.
  But those discussions can only happen in an environment of discretion and trust.
  The Chatham House Rule <https://www.chathamhouse.org/about-us/chatham-house-rule>`_ applies: you're welcome to use the information shared, but never reveal names, job titles, employers, or other identifying information about other participants outside of Slack, unless you've recieved their explicit permission to do so.
```

Reviewer: I think I'm actually down with this almost verbatim.
